### DISCUSSION TITLE
Script to run UI

### DISCUSSION THREAD

**Discussion by nathanhere** [open-webui/open-webui#1432] [General]
This might be a super obvious question, but what command launches the UI when running via docker compose? Been searching for the past hour 😅

**Comment by ddanat-smm** [comment]
To start:
docker compose up -d

To stop:
docker compose down

It's like that for pretty much all docker compose setups with seldom variation to it.

**Comment by ddanat-smm** [comment]
Disregard what I wrote, the docs are the best place to look:
https://docs.openwebui.com/getting-started/
